python/ray/util/placement_group.py

```diff
@@ -78,9 +78,18 @@ def bundle_count(self):
         return len(self.bundle_cache)
 
     def _get_none_zero_resource(self, bundle: List[Dict]):
+        # This number shouldn't be changed.
+        # When it is specified, node manager won't warn about infeasible
+        # tasks.
+        INFEASIBLE_TASK_SURPRESS_MAGIC_NUMBER = 0.0007
```

Comment:
```suggestion
        INFEASIBLE_TASK_SURPRESS_MAGIC_NUMBER = 0.010101
```

Reply:
Chose 0.0101. Seems like digits after that was cut.